.npmignore

```diff
@@ -4,3 +4,7 @@ npm-debug.log
 coverage
 .gitignore
 .DS_Store
+.npmignore
+__tests__
```

Comment:
That just talks about removing npat, which is not the same thing as removing the ability to publish tests.